src/components/MediaDetails/MediaDetailsVisual.js

```diff
@@ -58,30 +59,17 @@ function MediaDetailsVisual({ file, docType }) {
           </div>
         </div>
       </div>
-      <div>
-        <h3 className="font-medium text-charcoal-900">Information</h3>
-        <dl className="mt-2 border-t border-b border-charcoal-100 divide-y divide-charcoal-100">
-          {file?.id &&
-            Object.keys(file).map((key) => {
-              if (isDisplayablePropMedia(key, file[key])) {
-                return (
-                  <div
-                    key={key}
-                    className="py-3 flex justify-between text-sm font-medium"
-                  >
-                    <dt className="text-charcoal-500 capitalize">
-                      {key === 'mimeType' ? 'File Type' : key}
-                    </dt>
-                    <dd className="text-charcoal-900 truncate pl-8">
-                      {file[key]}
-                    </dd>
-                  </div>
-                )
-              }
-              return null
-            })}
-        </dl>
-      </div>
+      {MediaDetailsSettings({ file })}
```

Comment:
Given that MediaDetailsSettings is a functional component it should follow component conventions and be declared as such <MediaDetailsSettings file={file} />

Reply:
oh right!